.circleci/config.yml

```diff
@@ -47,13 +53,17 @@ commands:
   svn_create_tag:
     description: "Create a SVN tag"
     steps:
-      - set_verision_variable
-      - run: svn cp trunk tags/${VERSION}
+      - set_version_variable
+      - run:
+          command: |
+            svn up tags/${VERSION}
+            svn rm tags/${VERSION} &>/dev/null
```

Comment:
Without `&>/dev/null`, this will probably exit if there is no tag `tags/${VERSION}`

Reply:
Without `&>/dev/null`, this will probably exit if there is no tag `tags/${VERSION}`:

```sh
$ svn rm tags/32423431341  
svn: E155007: 'simple-social-icons/tags/32423431341' is not a working copy
```